bundle/porter.yaml

```diff
@@ -155,11 +165,22 @@ install:
       command: ./helpers.sh
       arguments:
         - deploy_secrets
+  - exec:
+      description: "Grab ArgoCD Initial Password"
+      command: ./helpers.sh
+      arguments:
+        - return_argo_initial_pass
+      # outputs:
+      #   - name: argo-initial-passwd
+      #     # this feels hackish. Porter has a limitation of needing to pass through
+      #     # regex/jsonrgex rather than just giving you the direct output of
+      #     # the command you want output from...
+      #     regex: '^.'
```

Comment:
Yes, because technically, I would prefer not to output to a file and instead grab the string from the output, but there is a problem with doing that. What I have in there is supposed to work but doesn't, so I would like to go back to that approach once Porter fixes it on their side.

Reply:
as in the regex comment is still valid, and it is still the solution to the problem that I feel is better than storing a file in the container, overall security wise it feels cleaner to extract and store in porter than extract and duplicate